### ISSUE TITLE
create multiple dhcp scopes

### ISSUE THREAD

**Issue by Simon-CR** [BeryJu/gravity#1071]
As I am adding multiple dhcp scopes, when I hit the create button the fields are populated with the previous scope that I've added, unless I reload the page.

not a deal breaker, just a small bug

**Comment by Simon-CR** [comment]
as a small extra note, if I change some of the fields and save, it'll update the "original" scope (similar to if I was editing the scope instead of pressing creating a new one

**Comment by BeryJu** [comment]
yeah this is somewhat of a known bug, this is due to most of the UI elements being borrowed from https://goauthentik.io/ which suffers the same issue, once it's fixed there it'll be fixed here
